### ISSUE TITLE
Missing topic:  Accessibility

### ISSUE THREAD

**Issue by andybeach** [w3c/webmediaguidelines#7]
the initial draft should include guidance on closed captions/subtitling as all customers will have some sort of requirement around it.

**Comment by mavgit** [comment]
I suggest generalizing this to support for [W3C Web Content Accessibility Guidelines](https://www.w3.org/WAI/intro/wcag) (which includes captions and a lot more).

**Comment by joelkorpi** [comment]
Are there any further thoughts on this or are we going to just reference the W3C Accessibility doc as mav suggested?

**Comment by mavgit** [comment]
As I stated in a meeting, I propose you include a reference to the W3C Web Content Accessibility Guidelines and include one related code example on captions and/or subtitles.
